frb/data/Galaxies/Literature/fong2021_photom.ascii

```diff
@@ -0,0 +1,2 @@
+|     Name |         ra |        dec | 2MASS_J | 2MASS_J_err | 2MASS_H | 2MASS_H_err | 2MASS_K | 2MASS_K_err |
```

Comment:
Just confirming -- these are *without* Galactic extinction corrections, right?

Reply:
Correct, these are not corrected for extinction